### DISCUSSION TITLE
The way to upgrade from v0.5.6

### DISCUSSION THREAD

**Discussion by dhenry123** [mealie-recipes/mealie#3023] [General]
Hi congratulations for this new release,
but how could i migrate from v0.5.6 to v1.0.0 ?
Best regards

**Comment by michael-genson** [comment]
Hey there!

We have a migration guide published here:

https://nightly.mealie.io/documentation/getting-started/migrating-to-mealie-v1/

**Reply by dhenry123** [reply]
I found it in the repo and I just came back to this discussion to indicate this same link :) 
Thanks again
